sklearn/tests/test_pipeline.py

```diff
@@ -564,6 +564,8 @@ def test_pipeline_slice():
     assert isinstance(pipe2, Pipeline)
     assert pipe2.steps == pipe.steps[:-1]
     assert 2 == len(pipe2.named_steps)
+    assert pipe2.memory == pipe.memory
+    assert pipe2.verbose == pipe.verbose
```

Comment:
These are the defaults so I assume the test would still pass on master. Let's give them non-default values?

Reply:
Thanks for the comment, fixed it now and I am now waiting for the tests to pass.